src/Microsoft.AspNetCore.ResponseCaching/Internal/HttpHeaderParsingHelpers.cs

```diff
@@ -0,0 +1,118 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Globalization;
+using Microsoft.Extensions.Primitives;
+
+namespace Microsoft.AspNetCore.ResponseCaching.Internal
+{
+    internal static class HttpHeaderParsingHelpers
+    {
+        private static readonly string[] DateFormats = new string[] {
```

Comment:
It is private in that repo. We are considering moving a few parts of this pr to abstractions so we'll see if this is still needed here.